src/geo/ui/widgets/time-series/histogram-view.js

```diff
@@ -0,0 +1,102 @@
+var $ = require('jquery');
+var _ = require('underscore');
+var Model = require('cdb/core/model');
+var View = require('cdb/core/view');
+var HistogramChartView = require('../histogram/chart');
+
+/**
+ * Time-series histogram view.
+ */
+module.exports = View.extend({
+
+  defaults: {
+    width: 400
+  },
+
+  initialize: function() {
+    _.bindAll(this, '_onWindowResize');
+    $(window).bind('resize', this._onWindowResize);
+
+    this.filter = this.options.filter;
+
+    this.viewModel = new Model({
+      width: this.defaults.width,
+      margins: { // TODO could be calculated from element styles instead of duplicated numbers here?
+        top: 0,
+        right: 24,
+        bottom: 0,
+        left: 24
+      },
+      histogramChartHeight:
+        48 + // inline bars height
+        20 + // bottom labels
+        4 // margins
+    });
+    this.add_related_model(this.viewModel);
+    this.viewModel.bind('change:width', this._onChangeWidth, this);
+  },
+
+  render: function() {
+    this.clearSubViews();
+    this._createHistogramView();
+    this._onWindowResize();
+    return this;
+  },
+
+  clean: function() {
+    $(window).unbind('resize', this._onWindowResize);
+    View.prototype.clean.call(this);
+  },
+
+  _createHistogramView: function() {
+    this.chartView = new HistogramChartView({
+      type: 'time',
+      animationSpeed: 100,
+      margin: {
+        top: 4,
+        right: 4,
+        bottom: 20,
+        left: 4
+      },
+      y: 0,
+      handles: true,
+      width: this._histogramChartWidth(),
+      height: this.viewModel.get('histogramChartHeight'),
+      data: this.model.getData()
```

Comment:
@javierarce There are some options here that perhaps could be defaults inside the chart (e.g. `y: 0` and `animationSpeed: 100`), instead of having to pass them explicitly for all instances?